company/okrs/2020_q1.md

```diff
@@ -0,0 +1,69 @@
+# 2020-Q1 OKRs
+
+## CEO
+
+1. **CEO: Grow ARR consistently**
+   1. CEO KR: [$N][N] net new ARR
+   1. CEO KR: Sales leadership owns revenue targets
+   1. CEO KR: Sales, engineering, and product have a successful process for prioritization
+1. **CEO: Aspirational, efficient all-remote team**
+   1. CEO KR: Roles and responsibilities are documented and maintained in handbook
+   1. CEO KR: Handbook-first
+
+## Sales and marketing
+
+1. **Sales (Closing Deals): Achieve [$N][N] net new ARR.**
+   1. Sales KR: Have each AE close 3 Tier 1 deals (>=$100K ARR) and 10 non-Tier 1 deals, 2 of which are from their own outbound outreach efforts.
+   1. Sales KR: Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Sales KR: Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
+   1. VP Business (Success) KR: Generate at least [$M][M] in net new ARR from existing accounts.
+   1. VP Business (Success) KR: Hire a Head of Customer Success who can own this function.
+   1. VP Business (Success) KR: Reach monthly NPS of 60 at all Tier 1 customers.
+   
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: WIP Deliver on product roadmap**
+   1. Product KR: Roadmap communicates rough timeline for upcoming work.
+   1. Product KR: Build out per-team strategy RFCs to buffer roadmap.
```

Comment:
@christinaforney 

- For per-team strategies, it would be nice for these to be more permanent and canonical. Can you add them to the handbook on the page for each team? For example, https://about.sourcegraph.com/handbook/engineering/distribution already has a vision statement, so putting a strategy RFC out would run the risk of diverging from that and from the linked Google Doc.

Also, the 2nd 2 KRs here are things to do, not results. Please reword them to describe an outcome (eg "Each team has ..." and "Each feature is ...").